servicecontrol/containerization/index.md

```diff
@@ -0,0 +1,171 @@
+
+## Containers overview
+
+ServiceControl is split into multiple docker images for each instance type. These are:
+
+* The error instance
+* The audit instance
+* The monitoring instance
+
+The images for each of these containers are further split into an init container and a runtime container. This means that there are 6 images for all instances of ServiceControl.
+
+## Init containers
+
+The init containers are used to create or upgrade the infrastructure required for ServiceControl and are based on the [Kubernetes init containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/). Once the init container has been run, it will shut down and the runtime container can be run. The runtime container will use the infrastructure that has been created and if the init container was not run the runtime container will fail.
+
+The init containers are similar to [installing all ServiceControl service types (regular, error, and monitoring) via the ServiceControl Management Utility or the Powershell Cmdlets](/servicecontrol/installation.md). This includes creating the database as well as the queues that ServiceControl uses and runtime containers are similar to starting the corresponding Windows Services.
+
+## Transports
+
+Each supported transport and topology for ServiceControl has further been broken down into it's own set of images. The following transport and topologies are available on dockerhub:
+
+* [Amazon SQS](https://hub.docker.com/search?q=servicecontrol.amazonsqs&type=image)
+* [Azure Service Bus](https://hub.docker.com/search?q=servicecontrol.azureservicebus&type=image)
+* [Azure Storage Queues](https://hub.docker.com/search?q=servicecontrol.azurestorageque&type=image)
+* [RabbitMQ Conventional Topology](https://hub.docker.com/search?q=servicecontrol.rabbitmq.conven&type=image)
+* [RabbitMQ Direct Topology](https://hub.docker.com/search?q=servicecontrol.rabbitmq.direct&type=image)
+* [Sql Server](https://hub.docker.com/search?q=servicecontrol.sqlserver&type=image)
+
+## Parameters
+
+All parameters that the ServiceControl instance supports can be provided by passing the parameter as an environment variable. E.g.
+
+`-e "ServiceControl/Port=12345"`
+
+The most common parameters used are likely to be:
+
+* ServiceControl/ConnectionString
+* ServiceControl/LicenseText
+* ServiceControl.Audit/ConnectionString
+* ServiceControl.Audit/LicenseText
+* ServiceControl.Audit/ServiceControlQueueAddress
+* ServiceControl/RemoteInstances
+* Monitoring/ConnectionString
+* Monitoring/LicenseText
+
+NOTE: For hosting in Azure Container Instances parameter names are supported that use only underscores, e.g. ServiceControl_Audit_LicenseText
+
+These parameters can also be added by using a standard Docker environment file. Every parameter has its own line and does not need to be enclosed by  quotes. It can then be used by Docker as follows:
```

Comment:
```suggestion
These parameters can also be added by using a standard Docker environment file. Every parameter has its own line and does not need to be enclosed by quotes. It can then be used by Docker as follows:
```